.github/CODEOWNERS

```diff
@@ -1 +1 @@
-.github/CODEOWNERS @SonarSource/languages-team-phpy
+.github/CODEOWNERS @SonarSource/languages-team-python
```

Comment:
I think the ownership of this repository should be transferred to @SonarSource/languages-team-cloud